tests/http/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Starlite is only available on python 3.8+
+try:
+
```

Comment:
I feel like there is a starlite import missing here. If not, could you explain where the `ModuleNotFoundError` is supposed to come from :eyes: I'm confused.

Reply:
Right, there a missing `import starlite`

Reply:
Looks like it's still missing :eyes: